lib/dpul_collections_web/components/search_bar_component.ex

```diff
@@ -22,11 +22,7 @@ defmodule DpulCollectionsWeb.SearchBarComponent do
                 placeholder={gettext("Search")}
                 dir="auto"
               />
-              <button
-                id="search-button"
-                type="submit"
-                class="btn-secondary px-4 h-8 invisible flex-none"
-              >
+              <button id="search-button" type="submit" class="btn-secondary px-4 h-8 flex-none">
```

Comment:
I would argue that the search button / search form is accessible the way it is, pending usability testing. It becomes visible when the input is focused. If there were a way to make the check pass without changing the behavior that would be nice. If not, we should consider whether this has implications for our design choices. And also probably remove the logic that controls its visibility, since that would be defunct.